README.md

```diff
@@ -45,7 +45,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://deploy-preview-5--musong-math-magicians.netlify.app/)
+- [Live Demo Link](#)
 
```

Comment:
- [Optional] Dear @ernestmusong, I see you miss understood the requested changes by the previous reviewer. Please for the live version, you should delete the live demo section if you decide not to deploy the application. If project requirements say you should deploy, then you should put the link to the live version `NOT TO YOUR GIT HUB REPO`.